### PR TITLE
mrboom 5.2

### PR DESCRIPTION
**Pull request by frranck**

### CODE REVIEW

File: Formula/mrboom.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Mrboom < Formula
   desc "Eight player Bomberman clone"
   homepage "http://mrboom.mumblecore.org/"
-  url "https://github.com/Javanaise/mrboom-libretro/releases/download/5.0/MrBoom-src-5.0.tar.gz"
-  sha256 "88bd9fee69a9d3a5f3edafde9cfb5f9a8ca415041281fc19c1e7bc2a75407829"
+  url "https://github.com/Javanaise/mrboom-libretro/releases/download/5.2/MrBoom-src-5.2.454d614.tar.gz"
```
Comment (carlocab) [inline Formula/mrboom.rb:4]:
```suggestion
  url "https://github.com/Javanaise/mrboom-libretro/releases/download/5.2/MrBoom-src-5.2.454d614.tar.gz"
  version "5.2"
```
The version isn't being parsed correctly from the `url`, so we need to specify this explicitly.

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks, @frranck. One suggestion. When you make this change, please squash it with your existing commit, and use the commit message
```
mrboom 5.2
```
For future reference, please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings. Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) or the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md) for more details.

**Comment by carlocab** [comment]
Weird; the tests were fine when we didn't specify `version`.

**Comment by carlocab** [comment]
Ok, now the tests on Big Sur succeeded, when they failed previously (with `version` specified). Very weird. Probs a race condition.

**Comment by carlocab** [comment]
Please try this:
```diff
diff --git a/Formula/mrboom.rb b/Formula/mrboom.rb
index ed1710887f..6d27fb9d95 100644
--- a/Formula/mrboom.rb
+++ b/Formula/mrboom.rb
@@ -27,12 +27,12 @@ class Mrboom < Formula
     require "expect"
     require "timeout"
     PTY.spawn(bin/"mrboom", "-m", "-f 0", "-z") do |r, _w, pid|
-      sleep 1
+      sleep 15
       Process.kill "SIGINT", pid
       assert_match "monster", r.expect(/monster/, 10)[0]
     ensure
       begin
-        Timeout.timeout(10) do
+        Timeout.timeout(30) do
           Process.wait pid
         end
       rescue Timeout::Error
```
Squash all your changes into one commit, please. (Still with the message `mrboom 5.2`.)

**Comment by carlocab** [comment]
By the way, when you squash your commits, you're supposed to have only a single commit, not three.

**Comment by frranck** [comment]
I can't squash until I merge, because I'm using the web UI. but then it will use the latest commit message.

**Comment by carlocab** [comment]
This can't be merged using the web UI because the bottle commit needs to be merged simultaneously. The bottle commit needs to be generated by BrewTestBot.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**